src/components/Audiences/processDestinations.js

```diff
@@ -18,41 +18,38 @@ import {
 } from "../../utils";
 
 export default ({ destinations, config, logger }) => {
-  const urlDestinations = destinations
-    .filter(dest => dest.type === "url")
-    .map(dest =>
-      assign(
-        {
-          id: dest.id
-        },
-        dest.spec
-      )
-    );
+  if (config.destinationsEnabled === undefined || config.destinationsEnabled) {
+    const urlDestinations = destinations
+      .filter(dest => dest.type === "url")
+      .map(dest =>
+        assign(
+          {
+            id: dest.id
+          },
+          dest.spec
+        )
+      );
 
-  if (
-    urlDestinations.length &&
-    (config.destinationsEnabled === undefined || config.destinationsEnabled)
-  ) {
-    fireDestinations({
-      logger,
-      destinations: urlDestinations
-    }).then(result => {
-      console.log(result);
-    });
-  }
-
-  const cookieDestinations = destinations
-    .filter(dest => dest.type === "cookie")
-    .map(dest => dest.spec);
-
-  cookieDestinations.forEach(dest => {
-    if (isNonEmptyString(dest.name)) {
-      cookie.set(dest.name, dest.value || "", {
-        domain: dest.domain || "",
-        expires: dest.ttl ? dest.ttl : 6 * 30 // default of 6 months
+    if (urlDestinations.length > 0) {
```

Comment:
Can we just do `if (urlDestinations.length) {` like you did in `processIdSyncs.js`?